go.mod

```diff
@@ -5,7 +5,7 @@ go 1.21
 require (
 	dario.cat/mergo v1.0.0
 	github.com/giantswarm/apiextensions-application v0.6.1
-	github.com/giantswarm/clustertest v0.20.0
+	github.com/giantswarm/clustertest v0.20.1-0.20240512235412-f3c924c97b9b
```

Comment:
TODO: update after merging and releasing https://github.com/giantswarm/clustertest/pull/181

Reply:
Updated.